.gitignore

```diff
@@ -15,3 +15,5 @@
 /.release
 .elixir.plt
 erl_crash.dump
+/lib/*/cover
+/lib/*/_build
```

Comment:
I think we need `make` target for coverage reporting to avoid fiddling with `PATH` and be consistent with what we have today for testing.